models/evaluator.py

```diff
@@ -0,0 +1,30 @@
+from nltk.translate.bleu_score import sentence_bleu
+
+
+def bleu_evaluation(prediction, truth, verbose=False):
+    """
+    evaluating a translation sentence against one or more reference sentences.
+    Args:
+        prediction (string): translated sentence
+        truth (list): list of reference sentences
+        verbose (boolean): to print or not to print BLEU scores
+    Returns:
+        nothings for now, just print BLEU scores
+    Example:
+        truth = [['i love cats'], ['i love hats']]
+        candidate = 'i love cat'
+        bleu_evaluation(candidate, truth)
+    """
+    print('translations=[%s], references=%s' % (prediction, truth))
```

Comment:
I think you should add verbose to this print, not to the other ones, since the amount of prediction and truth variables could be more than just one sentence it can be annoying to print all of them